src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Bulldozer(),
+                new Excavator(),
+                new Truck()
+        };
+        for (Machine broom: machines) {
```

Comment:
I know it looks funny but let's stick with a more formal name :)
And check how is broom translated into Ukrainian
```suggestion
        for (Machine machine: machines) {
```